### PR TITLE
Update the Readme

### PR DESCRIPTION
**Pull request by amirhaleem**
This PR updates the readme to contain a guide that should be helpful for someone building their own Hotspot-compatible device and adding it to the network.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,193 @@
...
+|/ip4/192.168.3.6/tcp/36397|/ip4/104.248.206.159/tcp/443 |/p2p/11wDt78AktL5ZMmV8ePxZWAu8wyVRLbArFeqhCVExzyd3HP6hQa |   {ok,"fluffy-sand-yeti"}    |
+|/ip4/192.168.3.6/tcp/36397|/ip4/192.168.2.134/tcp/62146 |/p2p/11xxCzYzQAxhfazMVa6vi7QBD4ct85baw7FLG9KPmERVttXBKhQ |{ok,"beautiful-coffee-oyster"}|
++--------------------------+-----------------------------+---------------------------------------------------------+------------------------------+
+```
+
+As long as you have an address listed in `listen_addrs` and some peers in the table at the bottom, you're connected to the p2p network and good to go.
+
+### Loading the Genesis block
+
+First, you need a genesis block from either the main network, or the Pescadero testnet. Get them here: [mainnet](NEED A URL HERE) or [pescadero](NEED A URL HERE).
```
Comment (madninja) [inline README.md]: needs actual URL to genesis block. Do we want to just put it on S3 somewhere?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,193 @@
...
+
+The `owner` is the address that receives mining rewards, and can make changes to the location of the gateway. In order to create the `sign_gateway` transaction, which has the gateway sign the first half of the transaction, either a separate `miner` or `blockchain-api` instance needs to be running or for the purposes of this guide we will assume that the `owner` is running the Helium Hotspot mobile app on either iOS or Android.
+
+First, get the address of the `gateway` via the CLI:
+
+```_build/prod/rel/miner/bin/miner peer addr```
+
+Then, craft the `sign_gateway` transaction as follows:
+
+```_build/prod/rel/miner/bin/miner ledger sign_gateway -g <gateway_address> -s <stake> -f <fee>```
```
Comment (madninja) [inline README.md]: this PR hasn't merged yet right?

File: README.md (hunk 1 above)
Comment (Vagabond) [reply README.md]: Yes

**Comment by evanmcc** [comment]
closing as stale

**Pull request closed without merging**